integrations/helm/chartsync/chartsync.go

```diff
@@ -98,85 +96,76 @@ func (chs *ChartChangeSync) Run(stopCh <-chan struct{}, errc chan error, wg *syn
 			select {
 			// ------------------------------------------------------------------------------------
 			case <-ticker.C:
-				fmt.Printf("\n\t... TICK at %s\n\n", time.Now().String())
+				fmt.Printf("\n\t... CHARTSYNC at %s\n\n", time.Now().String())
```

Comment:
Is this debug output?

Reply:
_passim_

Reply:
Yes, it is. I prefer to leave it here until we get to a stable phase.